### PR TITLE
chore: improve tests coverage #NTRN-309

### PR DESCRIPTION
**Pull request by swelf19**
The tests mostly about the keepers implementation.
Keepers coverage before the pr - 41.6%
After the pr - 87.3%
During the work 2 minor bugs were found - [#NTRN-317](https://p2pvalidator.atlassian.net/jira/software/projects/NTRN/boards/121/backlog?selectedIssue=NTRN-317), [#NTRN-318](https://p2pvalidator.atlassian.net/jira/software/projects/NTRN/boards/121/backlog?selectedIssue=NTRN-318)

### CODE REVIEW

File: x/feerefunder/keeper/keeper_test.go [hunk 1]
```diff
@@ -1,19 +1,35 @@
 package keeper_test
 
 import (
+	"fmt"
+	"strconv"
 	"testing"
 
+	types2 "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
```
Comment (mmikev) [inline x/feerefunder/keeper/keeper_test.go]: Let's rename `types2` to something more meaningful?

File: x/interchainqueries/keeper/process_block_results.go [hunk 2]
```diff
@@ -53,17 +56,19 @@ func checkHeadersOrder(header *tendermintLightClientTypes.Header, nextHeader *te
 	return nil
 }
 
+type Verifier struct{}
```
Comment (mmikev) [inline x/interchainqueries/keeper/process_block_results.go:59]: Could you clarify, please, why do we need these structs (`Verifier` and `TransactionVerifier`) now?

File: x/transfer/keeper/keeper_test.go [hunk 3]
```diff
@@ -0,0 +1,143 @@
+package transfer_test
+
+import (
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/errors"
+	types2 "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
```
Comment (mmikev) [inline x/transfer/keeper/keeper_test.go]: Let's rename this to something more meaningful

File: x/interchainqueries/keeper/process_block_results.go (hunk 2 above)
Comment (swelf19) [reply x/interchainqueries/keeper/process_block_results.go:59]: To simplify the code testing process by delegating a crypto verification to the structs we can mock

**Review by pr0n00gler — APPROVED** [review]

File: testutil/interchainqueries/keeper/interchainadapter.go [hunk 4]
```diff
@@ -42,9 +50,11 @@ func InterchainQueriesKeeper(t testing.TB) (*keeper.Keeper, sdk.Context) {
 		storeKey,
 		memStoreKey,
 		paramsSubspace,
-		nil, // TODO: do a real ibc keeper
-		nil, // TODO: do a real wasm keeper
-		nil,
+		ibcKeeper, // TODO: do a real ibc keeper
```
Comment (ratik) [inline testutil/interchainqueries/keeper/interchainadapter.go:53]: mb remove TODO

**Review by zavgorodnii — APPROVED** [review]

**Pull request merged**